### PR TITLE
add pipeline

### PR DESCRIPTION
**Pull request by falkben**
add sort by year script

### CODE REVIEW

File: sort_by_year.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    # copy files into year directory
+    match_str = filename[:13]
+    data_files = set(Path(working_path).glob(f"*{match_str}*"))
+    for data_file in data_files:
+        if len(argv) > 1 and "dryrun" in argv[1]:
+            print("moving", data_file, "to", target_path / data_file.name)
+        else:
+            # todo: change to:
+            data_file.rename(target_path / data_file.name)
+            # shutil.copy(data_file, target_path / data_file.name)
```
Comment (mdecleir) [inline sort_by_year.py:60]: If this line is not needed, please remove it.

**Review by mdecleir — CHANGES_REQUESTED** [review]
Just 2 minor comments.

File: sort_by_year.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# header_info.py: Script to print relevant information about the image files.
+# Marjorie Decleir
+# Updated (to Python 3.6) on 24-10-2018
```
[the remark this answers is not in the document]
Comment (falkben) [reply sort_by_year.py:3]: probably, but this is usually the kind of metadata i prefer to just have in commit messages (changes regarding a file).

as you probably figured out, this part was originally copied from the `headers_info.py`.

we should probably remove these types of comments and move anything relevant into the readme.

[the remark this answers is not in the document]
Comment (mdecleir) [reply sort_by_year.py:3]: I agree for the changes regarding the file. But the first line is a description of what the script is intended for. I find that useful to have.

**Pull request merged**